src/howdryiam/DRYWithASideOfLoops.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package howdryiam;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ *
+ * @author Rhonda Marshall
+ * Oct 4, 2015
+ */
+public class DRYWithASideOfLoops {
+
+    /**
+     * Driver to show how this problem could be encapsulated in a loop
+     */
+    public static void main() {
+        Scanner keyboard = new Scanner(System.in);
+        
+        // An ArrayList to store the LabelPrinter instances
+        ArrayList<LabelPrinter> labelPrinterList = new ArrayList<>();
```

Comment:
You should always complete the arraylist initialization with a type in all <> even after the new.